### PR TITLE
Tag usage update

### PR DESCRIPTION
**Pull request by Vandiver247**
Update to allow the usage of either tags or branch names during deployment.

### CODE REVIEW

File: zarf.yaml [hunk 1]
```diff
@@ -118,6 +129,12 @@ components:
   - name: setup
     description: "Required setup for the Zarf CDO package"
     required: true
+    # actions:
```
Comment (mikevanhemert) [inline zarf.yaml:132]: Recommend removing stale action

File: zarf.yaml [hunk 2]
```diff
@@ -15,6 +15,13 @@ variables:
     default: "main"
     prompt: true
```
Comment (mikevanhemert) [inline zarf.yaml:16]: Recommend adding description for this VAR

File: zarf.yaml [hunk 3]
```diff
@@ -15,6 +15,13 @@ variables:
...
 
+  - name: TAG
+    prompt: true
```
Comment (mikevanhemert) [inline zarf.yaml:19]: Recommend adding description for this VAR

File: zarf.yaml [hunk 4]
```diff
@@ -15,6 +15,13 @@ variables:
...
+
+  - name: REF
```
Comment (mikevanhemert) [inline zarf.yaml:21]: Remove unused var

File: zarf.yaml [hunk 5]
```diff
@@ -27,6 +34,10 @@ variables:
     default: "1"
     prompt: false
 
+  - name: DEPLOY_TYPE
```
Comment (mikevanhemert) [inline zarf.yaml:37]: Remove unused var

**Review by mikevanhemert — CHANGES_REQUESTED** [review]
Functionally verified deployment works as expected when setting the BRANCH or TAG. I love the simplicity. This is :chefs_kiss: beautiful

**Review by docandrew — APPROVED** [review]

**Review by docandrew — APPROVED** [review]

**Review by mikevanhemert — APPROVED** [review]

**Pull request merged**